firmware/controllers/actuators/idle_thread.cpp

```diff
@@ -130,7 +131,8 @@ percent_t IdleController::getRunningOpenLoop(float rpm, float clt, SensorResult
 	openLoopBase = running;
 
 	// Now we bump it by the AC/fan amount if necessary
-	openLoopAcBump = engine->module<AcController>().unmock().acButtonState ? engineConfiguration->acIdleExtraOffset : 0;
+	openLoopAcBump = (!hasAcPressure() || engine->module<AcController>().unmock().acPressureSwitchState)
```

Comment:
I think, this can/should just respect `acCompressorState`, as the A/C delay is documented as being with regard to idle control.